usagov/vote-gov-drupal
# VOTE-1064 basics block dynamic headings

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-1064

## Description

In the basics modules taxonomy, change the heading levels dynamically based on whether the basics block heading is set.

## Deployment and testing

### Pre-deploy

1. lando retune

### QA/Test

1. Sign into the CMS locally
2. Go to http://vote-gov.lndo.site/register and edit the basics block to have a heading.
3. Confirm the Basics Block heading is h2 and the module subheadings are h3.
4. Remove the Basics Block heading
5. Confirm the module subheadings have changed to h2
6. Repeat steps 2-5 for Spanish. You might have to create the new translation if it doesn't exist yet.

## Checklist for the Developer




## Checklist for the Peer Reviewers
